src/core/cubic.h

```diff
@@ -7,6 +7,12 @@
 
 #pragma once
 
+typedef enum HYSTART_STATE {
```

Comment:
Generally, I think we follow the convention of having `QUIC_` at the beginning of all types (enums included). Let's update this one.